components/MapSelection.jsx

```diff
@@ -112,7 +114,9 @@ class MapSelection extends React.Component {
         if (this.props.geomType === "DragBox") {
             this.drawInteraction = new ol.interaction.DragBox({
                 className: 'selection-drag-box',
-                condition: ol.events.condition.shiftKeyOnly
+                condition: () => {
+                    return ["ZoomIn", "ZoomOut"].includes(this.props.currentTask) || ol.events.condition.shiftKeyOnly;
+                }
```

Comment:
I think you can generally simplify the condition to `return true`, `shiftKeyOnly` looks like leftover code, and I don't think it makes any sense here

Reply:
I thought it was used by the shift+drag zoom already implemented but nope. I have removed all the condition